assets/src/scss/base/_palette.scss

```diff
@@ -21,5 +22,8 @@ $palette: (
 
   .has-#{$name}-background-color {
     background-color: $color;
+
+    --transparent-button--hover--color: #{$color};
```

Comment:
This is the only way I could think about in order to have the correct color on hover in those new buttons, however this will only work if the selected background color is from our own palette... It won't work if editors choose instead a color from the default WP palette, in that case for now I'm using a [dark grey as hover color](https://github.com/greenpeace/planet4-master-theme/pull/1651/files#diff-2bc1c6e089a7a01997bb250bca7b661343c35a1512e49d22a11b1f00ef03b783R300) to make sure that it's easy to read with the white background and still sort of fits in with our colors.